stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/BallerinaFileBuilder.java

```diff
@@ -96,6 +102,7 @@ public void build() {
             String filename = new File(fileDescriptorSet.getName()).getName().replace(PROTO_FILE_EXTENSION, "");
             String filePackage = fileDescriptorSet.getPackage();
             StubFile stubFileObject = new StubFile(filename);
+            StubFile servicestubFileObject = new StubFile(filename);
```

Comment:
Do we need to create an instance of StubFile? Shall we use ServiceFile to create service sample file.